docs/admin/install/troubleshoot.rst

```diff
@@ -25,6 +25,77 @@ This is a transient error that may affect any of the SecureDrop Workstation VMs.
 
+"Unable to reset PCI device"
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On some hardware, network devices (Ethernet and Wi-Fi) will not immediately work out of the box and require a one-time manual configuration on install. After Qubes starts for the first time, ``sys-net`` will fail to start:
+
+|screenshot_sys_net_pci_reset|
+
+Open a ``dom0`` terminal via **Q > Terminal Emulator**, and run the following command to list the devices connected to the ``sys-net`` VM.
+
+.. code-block:: sh
+
+  qvm-pci ls sys-net
+
+
+This will return the two devices (Ethernet and WiFi) that are connected to ``sys-net``:
+
+.. code-block:: sh
+
+  BACKEND:DEVID  DESCRIPTION                                                            USED BY
+  dom0:00_14.3   Network controller: Intel Corporation                                  sys-net
+  dom0:00_1f.6   Ethernet controller: Intel Corporation Ethernet Connection (5) I219-V  sys-net
+
+
+For both device IDs (e.g. ``dom0:00_1f.6`` and ``dom0:00_14.3``), you will need to detach and re-attach the device to ``sys-net``, then restart ``sys-net`` as follows:
+
+.. code-block:: sh
+
+  qvm-pci detach sys-net dom0:00_14.3
+  qvm-pci detach sys-net dom0:00_1f.6
+  qvm-pci attach sys-net --persistent --option no-strict-reset=True dom0:00_14.3
+  qvm-pci attach sys-net --persistent --option no-strict-reset=True dom0:00_1f.6
+  qvm-start sys-net
+
+
+``sys-net`` should now start, and network devices will be functional. This change is only required once on first install.  See the `Qubes documentation of this issue <https://www.qubes-os.org/doc/pci-troubleshooting/#unable-to-reset-pci-device-errors>`_ for more information.
+
+.. |screenshot_sys_net_pci_reset| image:: ../../images/screenshot_sys_net_pci_reset.png
+
+
+Full system freezes
+~~~~~~~~~~~~~~~~~~~
+
+A `known issue <https://github.com/QubesOS/qubes-issues/issues/8825>`_ with some hardware results in Qubes fully freezing.
+If you encounter this issue, you will need to forcibly restart your computer, usually by holding down the power button.
+
+When you boot up, you will see a black-and-white menu with the following options:
+
+.. code-block:: text
+
+  Qubes, with Xen hypervisor
+  Advanced options for Qubes (with Xen hypervisor)
+  UEFI Firmware Settings
+
+While ``Qubes, with Xen hypervisor`` is selected, press :kbd:`e` to edit the option. You should now see a rudimentary
+edit interface.
+
+Find the line that starts with ``multiboot2   /xen-`` and ends with ``${xen_rm_opts}``. Use the arrow keys to move your
+cursor to before ``${xen_rm_opts}`` and type :kbd:`cpufreq=xen:hwp=off` (leave a space between ``off`` and the ``$``.
+
+Press :kbd:`Ctrl-x` to continue with booting. This will fix the current boot, we now need to make the fix permanent.
+
+Once Qubes has started and you have logged in, open a ``dom0`` terminal via **Q > Terminal Emulator** and type
```

Comment:
Should this not be **Q > Settings Gear > Other Tools > Xfce Terminal** now?

Reply:
Yeah, it should. I just left it as is because the PCI section also needs the same fix and we wanted to try to use the gear emoji in https://github.com/freedomofpress/securedrop-workstation-docs/pull/222 but that doesn't work yet.

Will fix.